lib/brain.js

```diff
@@ -2727,6 +2749,35 @@ Brain.prototype._fiatReceipt = function _fiatReceipt () {
   })
 }
 
+Brain.prototype.areYouSureHandled = function areYouSureHandled (action) {
+  return _.includes(action, ARE_YOU_SURE_HANDLED) || 
+    (action === 'cancelPhoneNumber' && _.includes(this.complianceReason, ARE_YOU_SURE_HANDLED_SMS_COMPLIANCE))
+}
+
+Brain.prototype.areYouSure = function areYouSure () {
+  this._timedState('areYouSure')
```

Comment:
Should this have a timeout other than the default one?

Reply:
Default one is fine.